src/devices/Nmea0183/README.md

```diff
@@ -0,0 +1,34 @@
+﻿# NMEA 0183 Protocol
+
+## Summary
+
+NMEA stands for `National Marine Electronics Associations`.
```

Comment:
So this isn't really a binding then right? this is more of a standard for comunication. I'm fine with the way you have it, but what was the reasoning of having this separate to Neo-M8? Do we expect other bindings to use the same standard?

Reply:
@joperezr yes, I know @ZhangGaoxing has one which will likely work with that but some parameters might vary (i.e. NEO-M8 also supports I2c but I wasn't able to get it to work (it just didn't respond nor sent anything, also you have to set line endings on serial port etc)